lib/_stream_writable.js

```diff
@@ -293,22 +293,16 @@ function doWrite(stream, state, writev, len, chunk, encoding, cb) {
 }
 
 function onwriteError(stream, state, sync, er, cb) {
+  --state.pendingcb;
```

Comment:
Hm, so while I *think* this is correct, it feels unsafe to decouple `pendingcb` from actually calling the callback. We end up relying on `onwrite` to nextTick the next callback in the queue to ensure the order of events happens as planned.